### PR TITLE
fix: check for non-empty requirement config spec

### PR DESCRIPTION
**Pull request by ankitm123**
Signed-off-by: ankitm123 <ankitmohapatra123@gmail.com>

### CODE REVIEW

**Review by msvticket — APPROVED** [review]
That should work. Not that `&& !requirementsConfig.Spec.IsEmpty()` is actually needed now, but I suppose that could change.

File: OWNERS [hunk 1]
```diff
@@ -2,7 +2,9 @@ approvers:
 - rawlingsj
 - jstrachan
 - rajdavies
+- ankitm123
```
Comment (ankitm123) [inline OWNERS:5]: Also adding to owners file ...

**Pull request merged**